### ISSUE TITLE
[Bug]: Video page loads, but video wont play

### ISSUE THREAD

**Issue by MikkelHauge** [FreeTubeApp/FreeTube#5459]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

First off, video showing the issue, uploaded to imgur (clip is ~15 seconds long)

https://imgur.com/hZBw6rp

I load my subscriptions.

I click on a video

and whatever you'd call whats going on, in the video clip, is happening. Regardless of what video i click on. Trending videos, subscriptions etc.

It looks to drawing the page, then inserts the video, adds the frame for the whole video (instaed of thumbnail size) and then it keeps going back and forth with some annoyingly high speed.

I had issues with the video NOT playing at all (just nothing happening) earlier. I went backwards, and clicked the same video again, and it would work perfectly fine.

I've of course tried to restart Freetube multiple times.

sometimes when i then click, the video does go full size, but the video doesnt play, and instead it writes all this text, on top of the video-box:

DASH request error at URL: https://rr4---sn-4pcp5q3-g3xe.googlevideo.com/videoplayback?expire=1721862839&ei=VzahZqGgAoXi6dsPkfaAgAg&ip=77.33.137.38&id=o-AM_bTSvUDeK3lXOfbqzw79D0cgBMZNl-L_JAs5E0ERCx&itag=299&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278%2C298%2C299%2C302%2C303&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=Uw&mm=31%2C29&mn=sn-4pcp5q3-g3xe%2Csn-i5h7lnll&ms=au%2Crdu&mv=m&mvi=4&pcm2cms=yes&pl=21&initcwndbps=2026250&bui=AXc671IJ0cAhM_m93Yr8PJM5vkBrm_q7vtJD0xDr-_R6MikErJpUo5f3uJ1p2cs0-jrEbm5fNsQfduj_&spc=NO7bAWrwL-_QQNHGU0q4ORZ-aP7TjIk7MUkOHZ2VlAVT9tTZavNnUBGbpiQ7&vprv=1&svpuc=1&mime=video%2Fmp4&ns=Y0UZCxAaQzoJy__gFhCnaJMQ&rqh=1&gir=yes&clen=48848853&dur=105.388&lmt=1721830614597941&mt=1721840487&fvip=2&keepalive=yes&c=WEB&sefc=1&txp=6309224&n=xABu35j7adHcTDyBJL&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cxpc%2Cbui%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Crqh%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRQIhAKRhSxdV8G-bpF0zf2Rarhd5Gw8LKNRCfh4os_N7Mhu0AiA2eMAeX_11PPHyaBONBispstDeIKBiEHsX5uRMY5Nq2g%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AGtxev0wRQIhAPkKraGjADaOHkexZTnSoUCeRI_kfGegklLxCS5ty3kKAiAC3EwlNROuTfjfl7dkdQXNPyj8Wx1-E45n3EwnDadEYA%3D%3D&cver=2.20240111.09.00&cpn=EQF8NahE-1CAsOIH

basically a mix of this string or the blinky blinky

### Expected Behavior

Obviously i expected the video to just play. Like normal :)

### Issue Labels

content not loading

### FreeTube Version

v0.21.1 Beta 

### Operating System Version

Windows 11, version 10.0.22631

### Installation Method

.zip

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.21.1 Beta 

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by sushinateur** [comment]
Can confirm this behavior. 
Switching "Preferred API Backend" to "Invidious API" seems to be a workaround.

**Comment by OothecaPickle** [comment]
#5456

**Comment by MJBgood** [comment]
I am also facing this. The video fetch (to the path /videoplayback) is giving me a 403 Forbidden response. Is google blocking these requests?

**Comment by 4yman-0** [comment]
#5458 #5457 #5456

make sure you searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues.

**Comment by romamix** [comment]
Same issue. Switching to Invidious API with a non-default server helped.

**Comment by BlandBlatantRock** [comment]
Facing the same issue.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/5456

[Temporary workaround](https://github.com/FreeTubeApp/FreeTube/issues/5456#issuecomment-2248897869)
